test/celeritas/LArSphereBase.hh

```diff
@@ -45,6 +48,28 @@ class LArSphereBase : public GeantTestBase
         result.processes |= GeantImportDataSelection::optical;
         return result;
     }
+
+    SPConstOpticalPhysics build_optical_physics() override
```

Comment:
If we add another optical physics test, this should probably move to GeantPhysics base class with a flag for enabling optical physics (or a new OpticalGeantPhysics base...)